scripts/st2bootstrap-deb.sh

```diff
@@ -79,6 +79,10 @@ install_st2mistral() {
   /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf upgrade head
   # Register mistral actions
   /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf populate
+  
+  # Start Mistral
+  sudo service mistral-server start
+  sudo service mistral-api start
 }
```

Comment:
1) yup right
2) st2ctl worked, but it didn't support upstart correctly, well now it does https://github.com/StackStorm/st2/pull/2551
3) maybe, but if use in your script, make sure to
`sudo apt-get install -y sysvinit-utils` along with other stuff installations, it saves from awkward failures.